RocketChat/Rocket.Chat
# [FIX] Active users count on `@all` and `@here`

## Proposed changes (including videos or screenshots)



this PR updates the old `roomMembersCount` to count active users instead of everyone



## Issue(s)


## Steps to test or reproduce

Open rocket.chat 
Create a channel/room with all the members
Go to Administration > Settings > General > Notifications.
update Max Room Members Before Disabling All Message Notifications to 2  (if it matches the above requirements, if not, please use the number of active users you have in the room).
send a message with one of the active users and check if you've received a notification with the other one.


Expected Result:

Active users should've received the notification (except the one that send it).

## Further comments
